railties/lib/rails/code_statistics.rb

```diff
@@ -60,6 +63,14 @@ def self.register_directory(label, path, test_directory: false)
       self.test_types << label if test_directory
     end
 
+    # Add extensions to the output of the <tt>bin/rails stats</tt> command.
+    #
+    #   Rails::CodeStatistics.register_extension("txt")
+    def self.register_extension(extension)
+      self.extensions << extension
+      self.pattern = /^(?!\.).*?\.(#{self.extensions.join("|")})$/
```

Comment:
nitpick: It's better to use `Regexp.union`.

Reply:
@byroot Thank you for the comments.

Is this the kind of usage you had in mind for `Regexp.union`?
```suggestion
      self.pattern = Regexp.union(self.extensions.map { |ext| /^(?!\.).*?\.(#{Regexp.escape(ext)})$/ })
```
(It also added `Regexp.escape`.)

Reply:
More like: 

```suggestion
      self.pattern = /^(?!\.).*?\.(#{Regexp.union(extensions.map { |e| Regexp.escape(e) })})$/
```

NB: The `Regexp.escape` is a bit superfluous because I can hardly imagine an extension containing a special character, but might as well.

Reply:
Thanks. I have updated it.